### PR TITLE
Added docs for with/without

### PR DESCRIPTION
**Pull request by dha**
May want to change it to a =head2 heading, but for now have put it as a
=head3 under the =head2 for C<if>

### CODE REVIEW

File: lib/Language/control.pod [hunk 1]
```diff
@@ -238,6 +238,32 @@ two differences C<unless> works the same as L<if>:
 Implementation note: Currently, Rakudo will say "1 Nil 3 0" for the last
 example because it is not caught up to this part of the design yet.
 
+=head3 with, orwith, without
+
+The C<with> statement is like C<if> but tests for definedness rather than
```
Comment (vendethiel) [inline lib/Language/control.pod]: should `defined` be a link here?

File: lib/Language/variables.pod [hunk 2]
```diff
@@ -513,24 +513,25 @@ Therefore, the subroutine
 
 sub a {
     state @x;
-    @x.push(++$)
+    state $l = 'A'
+    @x.push($++)
```
Comment (vendethiel) [inline lib/Language/variables.pod]: shouldnt that be `$l`?

Comment (dha) [inline lib/Language/variables.pod]: Urk. Yeah, it should. I must have had a bad copy paste. Will fix. Thanks!

File: lib/Language/control.pod (hunk 1 above)
Comment (dha) [inline lib/Language/control.pod]: I don't think so, unless we have something to link to that's a description of definedness. I don't think it should be a link to C<defined> if that's what you're thinking.

**Pull request merged**